libr/core/cmd_debug.c

```diff
@@ -3615,6 +3615,13 @@ static int cmd_debug(void *data, const char *input) {
 			case '+':
 				r_debug_session_add (core->dbg, NULL);
 				break;
+			case '-':
+				if (input[3] == ' ') {
+					r_debug_session_delete (core->dbg, atoi (input + 3));
```

Comment:
Use r_num_math instead of atoi . So the user can use flags or math ops to express their numbers